src/plugins/downloader/main/download-manager-engine.ts

```
@@ -0,0 +1,432 @@
+   */
+  addToQueue(params: {
+    url: string;
+    title: string;
+    artist: string;
+    playlistFolder?: string;
+    trackId?: string;
+    isPlaylist?: boolean;
+    downloadFolder: string;
+    fileExtension: string;
+  }): string {
+    const { url, title, artist, playlistFolder, trackId, isPlaylist, downloadFolder, fileExtension } = params;
```

ESLint

[error] Replace `·url,·title,·artist,·playlistFolder,·trackId,·isPlaylist,·downloadFolder,·fileExtension` with `⏎······url,⏎······title,⏎······artist,⏎······playlistFolder,⏎······trackId,⏎······isPlaylist,⏎······downloadFolder,⏎······fileExtension,⏎···`

(prettier/prettier)